client/src/components/Modals/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+import "./style.css";
+import axios from 'axios';
+
+class MyVerticallyCenteredModal extends React.Component {
+  constructor(props) {
+    super(props)
+    this.logout = this.logout.bind(this)
+    this.state = {
+      modalShow: true
+    }
+  }
+
+  logout(event) {
+    event.preventDefault()
+    console.log('logging out')
+    axios.post('/logout').then(response => {
+      console.log(response.data)
+      if (response.status === 200) {
+        this.props.updateUser({
+          loggedIn: false,
+          username: null
+        })
+      }
+    }).catch(error => {
+      console.log('Logout error')
+    })
+  }
+  render() {
+    const loggedIn = this.props.loggedIn;
+    let modalClose = () => this.setState({ modalShow: false });
+    console.log(this.props);
+
+    return (
+      <div>
+        {loggedIn ? (
+          <Modal
```

Comment:
What's the difference between these two modals? If you extracted these out with a better component description and fed in the props to this you can know which modal is for what exactly. What if i wanted to reuse a modal somewhere else that is basically the same? You can then use these two different components else where as well.